lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -117,17 +333,22 @@ class window.Turbolinks
   @loadPage: (url, xhr, options = {}) ->
     triggerEvent 'page:receive'
     options.updatePushState ?= true
-
-    if doc = processResponse(xhr, options.partialReplace)
+    if upstreamDocument = processResponse(xhr, options.partialReplace)
       reflectNewUrl url if options.updatePushState
-      nodes = changePage(extractTitleAndBody(doc)..., options)
+      nodes = changePage(
```

Comment:
you can also do `gem 'turbograft', path: '../turbograft'` in your consumer, as long as you get the relative path correct

I _think_ you need to bundle install once this way, but after that, any on-the-fly modifications should be picked up iirc